app/view/twig/editcontent/fields/_repeater.twig

```diff
@@ -23,11 +23,11 @@
     <div class="repeater-slot">
         <script type="text/template">
             {% set index = '#' %}
-            {{ include('@bolt/editcontent/fields/_repeater-group.twig', { 'content': repeatfieldvals.getEmptySet , 'index': index}) }}
+            {{ include('@bolt/editcontent/fields/_repeater-group.twig', {'content': repeatfieldvals.getEmptySet, 'index': index}) }}
         </script>
 
         {% for index, repeatfieldset in repeatfieldvals %}
-            {{ include('@bolt/editcontent/fields/_repeater-group.twig', { 'content': repeatfieldset, 'index': index}) }}
+            {{ include('@bolt/editcontent/fields/_repeater-group.twig', {'content': repeatfieldset, 'index': index}) }}
```

Comment:
This is a change that moves away of what's outlined in #5232. For now, this is no big deal to me, but in the near future, i'd like someone (me? rarila?) to go over all of our twig templates and apply "CS for Twig", so to say.